### PR TITLE
Add Dance: Font-color dance

### PR DESCRIPTION
**Pull request by tnwjd4623**
Add font color dance using style color change

### CODE REVIEW

File: src/Dancer.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import fontSize, { reset as fontSizeReset } from './dances/font-size.js'
 import borderWidth, {
   reset as borderWidthReset,
 } from './dances/border-width.js'
+import fontColor, { reset as fontColorReset } from './dances/font-color.js'
 
 class Dancer {
   constructor() {
```
[the remark this answers is not in the document]
Comment (tnwjd4623) [reply src/Dancer.js]: I fixed it :)

**Comment by tnwjd4623** [comment]
I registered dance in constructor, deleted .idea dir and add my dance to README.md !

**Comment by tnwjd4623** [comment]
Sorry, I delete README_kr.md file. I will create new PR for README.

**Comment by Okazari** [comment]
I updated the code and gathered the commit into a new PR #111, no worry you're still the author of the commits :)

**Pull request closed without merging**